### PR TITLE
Move Conformance section into Introduction

### LINKED ISSUE #284: Align conformance classes with the controller document spec
Related to the resolution of https://github.com/w3c/controller-document/issues/7

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #284

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/vc-jose-cose/pull/299.html" title="Last updated on Sep 20, 2024, 8:20 PM UTC (2fa8fe9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/299/befdeae...selfissued:2fa8fe9.html" title="Last updated on Sep 20, 2024, 8:20 PM UTC (2fa8fe9)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
...
+            Requirements</a>.
+        </p>
+        <p>
+            Accordingly, Issuers, Holders, and Verifiers MUST understand the
+            JSON Web Token header parameter
+            <code>"alg": "none"</code> when securing [[VC-DATA-MODEL-2.0]]
+            with JSON Web Tokens.
+            When content types from [[VC-DATA-MODEL-2.0]] are secured using
+            JSON Web Tokens, the header parameter <code>"alg": "none"</code>,
+            MUST be used to communicate that a JWT Claims Set (a
```
Comment (TallTed) [inline index.html:272]:
```suggestion
            JSON Web Tokens, the header parameter setting <code>"alg": "none"</code>,
            MUST be used to communicate that a JWT Claims Set (a
```

File: index.html [hunk 2]
```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
...
+            When a JWT Claims Set (a Verifiable Credential or a
+            Verifiable Presentation) contains
+            <code>proof</code>, and the JSON Web Token header contains
+            <code>"alg": "none"</code>, the JWT Claims Set MUST be considered to
+            have no integrity protection.
+        </p>
+        <p class="advisement">
+            Verifiable Credentials and Verifiable Presentations are not
+            required to be secured or integrity protected or to contain a
+            <code>proof</code> member.
```
Comment (TallTed) [inline index.html:284]:
```suggestion
            required to be secured nor integrity protected, nor to contain a
            <code>proof</code> member.
```

File: index.html [hunk 3]
```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
...
+            JSON Web Token header parameter
+            <code>"alg": "none"</code> when securing [[VC-DATA-MODEL-2.0]]
+            with JSON Web Tokens.
+            When content types from [[VC-DATA-MODEL-2.0]] are secured using
+            JSON Web Tokens, the header parameter <code>"alg": "none"</code>,
+            MUST be used to communicate that a JWT Claims Set (a
+            Verifiable Credential or a Verifiable Presentation) has no
+            integrity protection.
+            When a JWT Claims Set (a Verifiable Credential or a
+            Verifiable Presentation) contains
```
Comment (TallTed) [inline index.html:276]: This parenthetical note suggests that `a JWT Claims Set` could only be `a Verifiable Credential or a Verifiable Presentation`, but that's not so. If the intent is to say that these sentences only apply to any `JWT Claims Set` that *is* `a Verifiable Credential or a Verifiable Presentation`, they should be reworded, something like this ... and similar changes should be applied throughout the document.
```suggestion
            MUST be used to communicate that a JWT Claims Set that comprises a
            Verifiable Credential or a Verifiable Presentation has no
            integrity protection.
            When a JWT Claims Set that comprises a Verifiable Credential or a
            Verifiable Presentation contains
```

File: index.html [hunk 4]
```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
...
+            have no integrity protection.
+        </p>
+        <p class="advisement">
+            Verifiable Credentials and Verifiable Presentations are not
+            required to be secured or integrity protected or to contain a
+            <code>proof</code> member.
+        </p>
+        <p>
+            Issuers, Holders, and Verifiers MUST ignore all JWT Claims Sets that
+            have no integrity protection.
```
Comment (TallTed) [inline index.html:288]:
```suggestion
            Issuers, Holders, and Verifiers of Verifiable Credentials and/or
            Verifiable Presentations MUST ignore all, and MUST NOT produce any,
            JWT Claims Sets that have no integrity protection.
```

File: index.html [hunk 5]
```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
...
+        </p>
+        <p>
+            The JWT Claim Names <code>vc</code> and <code>vp</code>
+            MUST NOT be present in any JWT Claims Set.
```
Comment (TallTed) [inline index.html:292]: The existing restriction appears meant to apply to *all* JWT Claims Sets, for which it ought to be part of the JWT RFC, not this separate specification. Thus --
```suggestion
            MUST NOT be present in any JWT Claims Set that comprises a
            Verifiable Credential or a Verifiable Presentation.
```

File: index.html [hunk 6]
```diff
@@ -198,6 +198,103 @@ <h2 id="section-introduction">Introduction</h2>
         asymmetric encryption algorithms.
     </p>
 
+  <section id="conformance">
+    <section class="normative">
+        <h2 id="conformance-classes">Conformance Classes</h2>
```
Comment (TallTed) [inline index.html:203]: It seems odd that `<section id="introduction">` is not set as `<section id="introduction" class="normative">... and then the `class="normative"` removed from the sub-sections it contains.

```suggestion
  <section id="conformance" class="normative">
    <section class="normative">
        <h2 id="conformance-classes">Conformance Classes</h2>
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by decentralgabe** [comment]
Changes incorporated. Open for > 1 week. Editorial approval. Merging.

**Pull request merged**